File: BECore/app/model/ProjectProgressReport_AttachDetail.js
/// <summary>
/// Model EXTJS ProjectProgressReport_AttachDetail. 
/// </summary>
Ext.define('QLDT.model.ProjectProgressReport_AttachDetail', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectProgressReport_AttachDetailID',
    idType: 'Guid',
    fields: [
		{ name: "ProjectProgressReportID", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
        { name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
    ]
});

File: MT.Model/Business/ProjectProgressReport_AttachDetail.cs
﻿using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectProgressReport_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectProgressReport_AttachDetail:EntityBase.
    /// </summary>
    public class ProjectProgressReport_AttachDetail : EntityBase
    {
        [Key]
        public Guid ProjectProgressReport_AttachDetailID { get; set; }
        public Guid ProjectProgressReportID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public Guid? FileResourceID { get; set; }
        public string Description { get; set; }
    }
    #endregion
}
